ir_cdk_stacks/in_aur_01/response_lambda.py

```diff
@@ -0,0 +1,50 @@
+import json
+import gzip
+import base64
+import logging
+import urllib3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+http = urllib3.PoolManager()
+
+def lambda_handler(event, context):
+    # Process cloudwatch event logs
+    cloudwatch_event = event["awslogs"]["data"]
+    decoded = base64.b64decode(cloudwatch_event)
+    decompressed = gzip.decompress(decoded)
+    data = json.loads(decompressed)
+    
+    # Get sources of attack
+    sources = []
+    log_events = data["logEvents"]
+    for log_event in log_events:
+        message = log_event["message"]
+        if "FATAL:  password authentication failed for user" in message:
+            tok = message.split(" ")[2]
+            source = tok.split(":")[1]
+            sources.append(source)
+    
+    # Log to cloudwatch        
+    print(sources)
+    
+    # Send data to slack channel
+    webhook_url = ''
```

Comment:
Need to find a way to input the webhook URL, maybe CLI?

Reply:
Should be ok to hardcode for now, unless you want a config file, where you can specify a common webhook url for slack and email across all IRs?

Reply:
Slack will actually scan the web for the webhook URL. Got an email from Slack saying one of my commits has the URL and they automatically invalidate the URL.